src/iap/shared/components/carousel/carousel.component.scss

```
@@ -0,0 +1,94 @@
+@import '/src/scss/constants.scss';
+@import '/src/scss/mixins.scss';
+
+$iap-button-margin: 50px;
+
+//TODO: make it prettier, add animations
+
+.iap-carousel {
+	position: relative;
+	width: 100%;
+	height: 100%;
+
+	&__slide-container {
+		background-color: red;
+		width: 100%;
+		height: 100%;
+	}
+
+	&__prev-button,
+	&__next-button {
+		font-size: 48px;
+		font-weight: 600;
+		color: white;
+		user-select: none;
+
+		height: 100%;
+		display: flex;
+		position: absolute;
+		flex-direction: column;
+		justify-content: center;
+		bottom: 0;
+	}
+
+	&__prev-button:hover,
+	&__next-button:hover {
+		cursor: pointer;
+	}
+
+	&__prev-button {
+		left: -$iap-button-margin;
+
+		&-label {
+			padding: 20px 5px;
+			background: linear-gradient(90deg, $base-color, $second-color);
+		}
+		//TODO: Pick another color instead of gray
+		&-label:hover {
+			background: linear-gradient(90deg, gray, $second-color);
+		}
+	}
+
+	&__next-button {
+		right: -$iap-button-margin;
+
+		&-label {
+			padding: 20px 5px;
+			background: linear-gradient(-90deg, $base-color, $second-color);
+		}
+
+		&-label:hover {
+			background: linear-gradient(-90deg, gray, $second-color);
+		}
+	}
```

Just a suggestion, but it's easier for me to find some styles if they are written like here:
```suggestion
	&__prev-button {
		left: -$iap-button-margin;
	}
	&__prev-button-label {
		padding: 20px 5px;
		background: linear-gradient(90deg, $base-color, $second-color);
		
		//TODO: Pick another color instead of gray
		&:hover {
			background: linear-gradient(90deg, gray, $second-color);
		}
	}
	&__next-button {
		right: -$iap-button-margin;
	}
	&__next-button-label {
		padding: 20px 5px;
		background: linear-gradient(-90deg, $base-color, $second-color);
		
		&:hover {
			background: linear-gradient(-90deg, gray, $second-color);
		}
	}
```